docs/quickstarts/chrome-extension.mdx

```diff
@@ -0,0 +1,252 @@
+          </main>
+        </div>
+      </ClerkProvider>
+    )
+  }
+
+  export default IndexPopup
+  ```
+
+  ## Create a consistent CRX ID for your extension
+
+  Chrome Extensions have a unique CRX ID that rotates by default, which can cause errors and issues with the Clerk integration. To avoid these problems, ensure that you have a **consistent** CRX ID in both development and production for your extension by following these steps:
+
+  1. Visit Plasmo Itero's [Generate Keypairs](https://itero.plasmo.com/tools/generate-keypairs) tool.
+  1. Select the **Generate KeyPairs** button.
+  1. Save the private key somewhere safe in case you need it in the future. Save the public key for use in the next step, and the CRX ID for use in a later step.
+
+  ### Create an `.env.chrome` file to store your public key
+
+  Create an `.env.chrome` file and add your public key to it.
+
+  ```bash {{ filename: '.env.chrome' }}
+  CRX_PUBLIC_KEY=<PUBLIC KEY>
+  ```
+
+  ### Edit your `package.json` to use the new public key
+
+  If you were building an extension from scratch or using tools, you would need to maintain a `manifest.json` file, and potentially modify it for development and production. Plasmo uses the `package.json` to generate a `manifest.json` on build, and allows for the use of environment variables in `package.json`.
+
+  In your `package.json`, add `"key": "$CRX_PUBLIC_KEY"` to the `manifest` object, as shown in the following example:
+
+  ```bash {{ filename: 'package.json', mark: [33] }}
+  {
+    "name": "chrome-extension",
+    "displayName": "Chrome extension",
+    "version": "0.0.1",
+    "description": "A basic implementation of Clerk with a Plasmo-based Chrome Extension.",
+    "author": "Clerk",
+    "scripts": {
+      "dev": "plasmo dev",
+      "build": "plasmo build",
+      "package": "plasmo package"
+    },
+    "dependencies": {
+      "@clerk/chrome-extension": "1.1.5-snapshot.va3e1056",
+      "plasmo": "0.88.0",
+      "react": "18.2.0",
+      "react-dom": "18.2.0",
+      "tailwindcss": "3.4.1"
+    },
+    "devDependencies": {
+      "@ianvs/prettier-plugin-sort-imports": "4.1.1",
+      "@types/chrome": "0.0.258",
+      "@types/node": "20.11.5",
+      "@types/react": "18.2.48",
+      "@types/react-dom": "18.2.18",
+      "postcss": "8.4.33",
+      "prettier": "3.2.4",
+      "typescript": "5.3.3"
+    },
+    "manifest": {
+      "host_permissions": [
+        "https://*/*"
+      ],
+      "key": "$CRX_PUBLIC_KEY"
+    }
+  }
+  ```
+
+  ## Load your Chrome Extension into your Chromium-based browser
+
+  The CRXJS plugin that you installed facilitates Chrome Extension development by automatically "hot loading" the application while it runs. This ensures the `build/chrome-mv3-dev` folder remains up to date. Without the plugin, you would need to manually execute the build command and reload your Chrome Extension after each change. Plasmo automates this process, streamlining development.
```

Comment:
The guide never mentions installing this plugin - I thought Plasmo was going to be used instead?

Reply:
Yup! I missed this -- will update.

Reply:
okay once that's all correctly updated I can finish testing and reviewing!